contracts/astroport/oracle/src/error.rs

```diff
@@ -19,6 +19,12 @@ pub enum ContractError {
     #[error("Asset infos are not set")]
     AssetInfosNotSet {},
 
+    #[error("Asset infos are set")]
+    AssetInfosSet {},
```

Comment:
AssetInfosAlreadySet?

Reply:
yeah, that sounds a bit better, but tbh I don't want to spend time on it :) the current version is good enough

Reply:
done